packages/module-loader-commonjs/src/__tests__/index.test.ts

```diff
@@ -52,4 +59,30 @@ describe('createCommonJsModuleLoader', () => {
 
     expect(loadedModule).toEqual(expectedModule);
   });
+
+  it('sets agent if process.env.HTTP_PROXY is defined', async () => {
+    const currentHttpProxy = process.env.HTTP_PROXY;
```

Comment:
Hi @unstubbable ,
Thanks for your feedback. Imho it's better not to change global settings from within a test to avoid side effects. For now, there is no side effect, of course. Still feels better to me. What do you think?

Reply:
This will be gone anyways with the other change.

Reply:
Hi @unstubbable the tests are removed